src/lib/modules/scheduler/scheduler_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"time"
 
-	config "github.com/bench-routes/bench-routes/src/lib/config_v2"
+	config "github.com/bench-routes/bench-routes/src/lib/config"
```

Comment:
We need to update the Execute() function and also the respective test config file.

Reply:
Please cover this in your next commit.